docs/12-performance-test.md

```diff
@@ -33,25 +42,58 @@ Sleeper.
 Now find the ECS cluster that runs compaction tasks. It will be named `sleeper-${ID}-merge-compaction-cluster`.
 Click on one of the tasks and find the corresponding Cloudwatch log group. Use the following command in Cloudwatch
 Log Insights with the above log group selected:
+
 ```
 fields @message | filter @message like "compaction read"
 # If you want to see all the results, comment out the next two lines
 | parse @message '* * - * at * per second' as class, level, msg, rate
 | stats avg(rate)
 ```
+
 This should result a single value summarising the performance of the containers that are performing a compaction. See
 the table below for the results for various versions of Sleeper.
 
+## Compaction performance
+
+This test will continue running and wait for each operation in the tests to run. This will take around an hour. This
+can be used to measure performance of compaction in a way that controls for quirks in the ordering of partition
+splitting and compaction job creation.
```

Comment:
Should `in a way that controls for quirks` be `in a way that accounts for quirks` here? It doesn't read quite right

Reply:
I've rewritten that paragraph, thanks.